### PR TITLE
Fix design in signup steps

### PR DESCRIPTION
**Pull request by vinit717**
### Issue
#647 

Parent ticket: https://github.com/Real-Dev-Squad/todo-action-items/issues/178

 **Description**
Fix CSS for different media queries for signup steps

https://github.com/Real-Dev-Squad/website-www/assets/111434418/742cbe10-4f18-41e8-a806-2b8097265d66

### CODE REVIEW

File: app/styles/onboarding-card.module.css [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
-    margin-top: 4rem;
-    padding: 0 1rem;
+    padding: 2rem;
     box-shadow: 0px 4px 10px -2px var(--color-blackshadow), 0px 6px 15px -2px var(--color-blackshadow2);
+    margin-top: 4rem;
+}
+.logo__github-rds {
+    height: 5rem;
+    position: absolute;
+    top: -2.5rem
```
Comment (sumitd94) [inline app/styles/onboarding-card.module.css:17]: Dont use `-` here. Use translateX,Y (-50%)
(As discussed)

File: app/styles/onboarding-card.module.css [hunk 2]
```diff
@@ -77,4 +89,17 @@
         width: 2.8rem;
         height: 2.5rem;
     }
+
+    .btn-generateUsername{
+        width: 5rem;
+        font-size: 12px;
+        right: -4px;
```
Comment (sumitd94) [inline app/styles/onboarding-card.module.css:96]: why `-`?

File: app/styles/onboarding-card.module.css (hunk 1 above)
Comment (vinit717) [reply app/styles/onboarding-card.module.css]: done

File: app/styles/onboarding-card.module.css (hunk 2 above)
Comment (vinit717) [reply app/styles/onboarding-card.module.css]: done

File: app/styles/onboarding-card.module.css [hunk 3]
```diff
@@ -36,20 +49,21 @@
...
-    padding-top: 5rem;
+    padding-top: 3rem;
 }
 
-@media (max-width: 800px) {
+@media (max-width: 1024px) {
     .onboarding-card {
-        min-width: 60vh;
+        width: 80%;
+        padding: 2rem;
```
Comment (shubhamsigdar1) [inline app/styles/onboarding-card.module.css:58]: Why you added again `padding: 2rem`
already here: https://github.com/Real-Dev-Squad/website-www/pull/648/files#diff-bb9f4c5344f6ff1505af2885cdb84f11eb9ecb1d578d25ee809b6dba3cb75c30R10

File: app/styles/onboarding-card.module.css [hunk 4]
```diff
@@ -36,20 +49,21 @@
...
     line-height: 24px; 
-    padding-top: 5rem;
+    padding-top: 3rem;
 }
 
-@media (max-width: 800px) {
+@media (max-width: 1024px) {
     .onboarding-card {
-        min-width: 60vh;
+        width: 80%;
```
Comment (shubhamsigdar1) [inline app/styles/onboarding-card.module.css:57]: Why you written width only
not max-width and 
Nit: what is difference between `max-width` and `width`

File: app/styles/onboarding-card.module.css [hunk 5]
```diff
@@ -77,4 +91,16 @@
...
+    .btn-generateUsername{
+        width: 5rem;
+        font-size: 12px;
+    }
+}
+
+@media (max-width: 480px) {
+    .onboarding-card{
+        max-width: 90%;
+        padding: 3.5rem 1rem;
```
Comment (shubhamsigdar1) [inline app/styles/onboarding-card.module.css:104]: Why you added here again `padding: 3.5rem 1rem;`
already here: https://github.com/Real-Dev-Squad/website-www/pull/648/files#diff-bb9f4c5344f6ff1505af2885cdb84f11eb9ecb1d578d25ee809b6dba3cb75c30R67

File: app/styles/stepper.module.css [hunk 6]
```diff
@@ -99,8 +105,10 @@
   .preview__label {
     font-size: 1.125rem;
   }
+}
 
+@media (max-width: 480px) {
   .user-input {
-    width: 20rem;
+    width: 95%;
```
Comment (shubhamsigdar1) [inline app/styles/stepper.module.css:112]: Nit: here you used `rem` https://github.com/Real-Dev-Squad/website-www/pull/648/files#diff-20059dd1a6ac67594a0a67d38caf2a0ebe8b580d32c7eb0c72545ba70c4d92eaR94
and here you user `percentage`

Is there any reason

**Review by shubhamsigdar1 — COMMENTED** [review]
Thank you @vinit717 for making changes...seen video great work
written some comment can you go through once

File: app/styles/onboarding-card.module.css (hunk 3 above)
Comment (vinit717) [reply app/styles/onboarding-card.module.css:58]: I haven't add or remove

File: app/styles/onboarding-card.module.css (hunk 4 above)
Comment (vinit717) [reply app/styles/onboarding-card.module.css:57]: Width sets the exact width of an element, while max-width restricts it from exceeding a certain value

File: app/styles/onboarding-card.module.css (hunk 5 above)
Comment (vinit717) [reply app/styles/onboarding-card.module.css:104]: To make it more aligned in different screen sizes

File: app/styles/stepper.module.css (hunk 6 above)
Comment (vinit717) [reply app/styles/stepper.module.css:112]: To give the size I want to use %

Comment (shubhamsigdar1) [reply app/styles/stepper.module.css:112]: give more context...not understand

File: app/styles/onboarding-card.module.css (hunk 3 above)
Comment (shubhamsigdar1) [reply app/styles/onboarding-card.module.css:58]: give more context...not understand

File: app/styles/stepper.module.css (hunk 6 above)
Comment (vinit717) [reply app/styles/stepper.module.css:112]: for mobile view, I don't want to give specific width as in mobile very pixel count to give 95% of it's parent width

File: app/styles/onboarding-card.module.css (hunk 3 above)
Comment (vinit717) [reply app/styles/onboarding-card.module.css:58]: to make it more systematic I removed old padding and just gave 2rem

**Review by shubhamsigdar1 — APPROVED** [review]

**Pull request merged**